models/iaf_psc_alpha.h

```diff
@@ -29,7 +29,7 @@
 #include "event.h"
 #include "nest_types.h"
 #include "recordables_map.h"
-#include "ring_buffer.h"
+#include "ring_buffer_impl.h"
```

Comment:
Why include the `impl` file here? The main point of `impl` files is to avoid circular inclusion problems and code bloat. So unless absolutely unavoidable, I would suggest to include just `ring_buffer.h` here and include `ring_buffer_impl.h` only in the `.cpp` file.

Reply:
Thanks, done.